yt_dlp/extractor/startrek.py

```diff
@@ -0,0 +1,51 @@
+from .common import InfoExtractor
+from ..utils import int_or_none, urljoin
+
+
+class StarTrekIE(InfoExtractor):
+    _VALID_URL = r'(?P<base>https://(?:intl|www)\.startrek\.com)/videos/(?P<id>[^/]+)'
```

Comment:
this will fail for links without https (won't even work for http)

Reply:
That was somewhat intentional, because the site explicitly redirects you to `https` anyway and therefore I doubt a lot of people will run yt-dlp on a `http` URL, but if the convention is to use `https?` instead, sure, I’ll change it.

Reply:
yeah better to have as https?
Bcoz in the current state, url will be passed to generic -> redirect -> ur extractor.

it's better to not redirect.

Reply:
Fixed in 664b4c6.